docs/proposals/001-proposal-template.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Neat-enhancement-idea
+authors:
+  - "@sbose78"
+reviewers:
+  - "@alexmt"
+  - TBD
```

Comment:
The idea is that TBD will be suggested during review?

Reply:
Correct. The author may not know who the "component experts" would be.